cluster/node-pools/master-default/userdata.yaml

```diff
@@ -210,6 +211,17 @@ write_files:
 {{- if eq .Cluster.ConfigItems.teapot_admission_controller_process_resources "true" }}
             - --pod-process-resources
 {{- end }}
+{{- if eq .Cluster.ConfigItems.teapot_admission_controller_inject_environment_variables "true" }}
+            - --pod-enable-env-inject
+            - --pod-inject-env=_PLATFORM_ACCOUNT={{ .Cluster.Alias }}
+            - --pod-inject-env=_PLATFORM_OPENTRACING_TAG_ACCOUNT={{ .Cluster.Alias }}
+            - --pod-inject-env=_PLATFORM_OPENTRACING_LIGHTSTEP_COLLECTOR_PORT=8443
+            - --pod-inject-env=_PLATFORM_OPENTRACING_LIGHTSTEP_COLLECTOR_HOST=tracing.stups.zalan.do
+            - --pod-inject-env=_PLATFORM_OPENTRACING_LIGHTSTEP_ACCESS_TOKEN={{ .ConfigItems.lightstep_token }}
+{{- if eq .Cluster.Environment "e2e" }}
+              - --pod-inject-env=_PLATFORM_E2E=injected
```

Comment:
indentation looks wrong here

Reply:
fixed - strangely it hasn't let to an error